### DISCUSSION TITLE
Tengo problemas para ejecutar con maven

### DISCUSSION THREAD

**Discussion by SergitAlesandra** [community/community#143024] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Issues

### Body

al ejecutar con maven me dice que no tengo test para ejecutar estoy usando screemplay, gherkin, cucumber, intelij y me arroja el siguiente error [INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- clean:3.2.0:clean (default-clean) @ proyecto ---
[INFO] Deleting C:\Users\usuariop\Cproyecto\target
[INFO] 
[INFO] --- failsafe:3.0.0-M5:integration-test (default-cli) @proyecto---
[INFO] No tests to run.
[INFO] 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by samus-aran** [comment]
Hi @SergitAlesandra , thanks for participating!

We take our  [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
